config/mongoid.yml

```diff
@@ -10,7 +10,7 @@ development:
 test:
   clients:
     default:
-      uri: mongodb://localhost/licence_finder_test
+      uri: <%= ENV["MONGODB_URI"] || "mongodb://localhost/licence_finder_test" %>
```

Comment:
We should change this so the dev DB is separate to the test DB - the URIs in the environment should be different/separate.